<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>secondWindow</class>
 <widget class="QWidget" name="secondWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Encyclo</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>631</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/img/background.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../Users/Managua Leo/Desktop/Без имени-3.jpg</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="newgameButton">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>300</y>
     <width>130</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>130</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>130</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Bell MT</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.0738636 rgba(139, 18, 18, 255), stop:0.488636 rgba(0, 0, 0, 255), stop:0.931818 rgba(139, 23, 23, 255));</string>
   </property>
   <property name="text">
    <string>New game</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>535</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pediaButton">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>381</y>
     <width>140</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Bell MT</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.0738636 rgba(139, 18, 18, 255), stop:0.488636 rgba(0, 0, 0, 255), stop:0.931818 rgba(139, 23, 23, 255));</string>
   </property>
   <property name="text">
    <string>Encyclopedia</string>
   </property>
  </widget>
  <widget class="QPushButton" name="questButton">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>381</y>
     <width>140</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Bell MT</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.0738636 rgba(139, 18, 18, 255), stop:0.488636 rgba(0, 0, 0, 255), stop:0.931818 rgba(139, 23, 23, 255));</string>
   </property>
   <property name="text">
    <string>Quests</string>
   </property>
  </widget>
  <widget class="QPushButton" name="achievButton">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>432</y>
     <width>140</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>130</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Bell MT</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.0738636 rgba(139, 18, 18, 255), stop:0.488636 rgba(0, 0, 0, 255), stop:0.931818 rgba(139, 23, 23, 255));</string>
   </property>
   <property name="text">
    <string>Achievements</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rankButton">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>432</y>
     <width>140</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>130</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Bell MT</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.0738636 rgba(139, 18, 18, 255), stop:0.488636 rgba(0, 0, 0, 255), stop:0.931818 rgba(139, 23, 23, 255));</string>
   </property>
   <property name="text">
    <string>Rankings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="shopButton">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>483</y>
     <width>140</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>130</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Bell MT</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.0738636 rgba(139, 18, 18, 255), stop:0.488636 rgba(0, 0, 0, 255), stop:0.931818 rgba(139, 23, 23, 255));</string>
   </property>
   <property name="text">
    <string>Shop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="menuButton">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>483</y>
     <width>140</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>130</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Bell MT</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.0738636 rgba(139, 18, 18, 255), stop:0.488636 rgba(0, 0, 0, 255), stop:0.931818 rgba(139, 23, 23, 255));</string>
   </property>
   <property name="text">
    <string>Menu</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
